File: backend/art-galery/src/main/java/com/unmsm/movil/tecnisis/art_galery/infrastructure/adapters/input/rest/model/response/ArtisticEvaluationResponse.java
package com.unmsm.movil.tecnisis.art_galery.infrastructure.adapters.input.rest.model.response;

import lombok.*;

import java.math.BigDecimal;
import java.time.LocalDate;

@Builder @Getter @Setter
@NoArgsConstructor @AllArgsConstructor
public class ArtisticEvaluationResponse {
    private Long id;
    private LocalDate evaluationDate;
    private BigDecimal rating;
    private String result;
    private String status;
    private SpecialistResponse specialist;
    private RequestResponse request;
    private DocumentResponse document;
}

File: tecnisis-app/app/src/main/java/com/example/tecnisis/data/evaluations/ArtisticEvaluationResponse.kt
package com.example.tecnisis.data.evaluations

import com.example.tecnisis.data.document.DocumentResponse
import com.example.tecnisis.data.specialist.SpecialistResponse
import com.google.gson.annotations.SerializedName

data class ArtisticEvaluationResponse(
    val id: Long,
    @SerializedName("evaluation_date")
    val evaluationDate: String,
    val rating: Int,
    val result: String,
    val status: String,
    val specialist: SpecialistResponse,
    val document: DocumentResponse
)
